config/routes.rb

```diff
@@ -1,7 +1,12 @@
 Rails.application.routes.draw do
+
+  namespace :admin do
+    resources :items, except: [:destroy]
+  end
+
   resource :user, except: [:update]
-  resource :admin
   patch "/user", to: "users#update", as: :update_user
   get "login", to: "sessions#new"
   post "login", to: "sessions#create"
+  get 'admin/dashboard', to: 'admin/dashboard#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.